debian/rules

```diff
@@ -67,7 +67,7 @@ clean:
 	  ltmain.sh missing
 	rm -f configure Makefile.in man/Makefile.in src/Makefile.in
 	rm -f src/acconfig.h.in
-	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph-base/etc/logrotate.d/ceph.logrotate debian/radosgw.logrotate
+	rm -f debian/ceph-base.init debian/radosgw.init debian/ceph.logrotate debian/radosgw.logrotate
```

Comment:
Yeah, I guess we'd gotten it wrong in the earlier commit (0cbe3dea69604730bc7dba7bc3844294f4c9ac5a) - sorry for leading you astray!